src/ResourceManager/Profile/Commands.Profile.Test/ResourcGroupCompleterUnitTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.Azure.Commands.Profile.Test
+{
+    public class ResourceGroupCompleterUnitTests
+    {
+        [Fact]
```

Comment:
To test completely, we should think about throwing an exception from the client, verifying the behavior.

Reply:
Done